docs/blog/2022-08-12-observing-randomness.md

```diff
@@ -0,0 +1,140 @@
+
+The ultimate target of the LoE with regard to ceremony operations is to be able to carry out “async ceremonies”. In this setup, a timeframe will be set up for all operators executing the appropriate commands without everyone having to be online at the same time. This will become more important as the drand network grows and more participants in different timezones join the league. A monitoring tool that provides an accurate view of node and network state, as well as the operations that are being carried out is clearly an absolute necessity.
+
+## Monitoring architecture requirements
+
+To properly monitor the drand network we have some requirements:
+
+**Allow any LoE member to create their own monitoring infrastructure**
+
+We did not want to mandate using a specific metrics aggregation platform (e.g. Prometheus or InfluxDB), instead allowing for each organization to select the one that best meets their needs or with which they are most familiar.
+
+**Ease of deployment**
+
+We did not want to burden operators with complex deployment strategies, so we decided to modify the drand binary itself instead of adding a new binary to export the metrics.
+
+**Minimize the need for opening network ports**
+
+Since drand nodes are run by multiple organizations on their own networks, any changes to the firewall configuration increases friction and reduces the probability of success. Ideally no network ports other than the one used for generating randomness should need to be opened. At the very least no more incoming ports should be needed.
+
+**Use industry-standard formats where possible**
+
+To decrease the ramp-up time for integrating into an organization’s monitoring system, we need to use an industry-standard format to publish metrics.
+
+**Ability to see the state of the whole network from any point in the network**
+
+To allow all LoE members to create their own dashboards and alerts, we need to enable each node to provide a full view of the whole network.
+
+**(Close to) real-time view of the network state**
+
+Especially during ceremonies, it is important to see changes in node state as soon as possible (< 1 minute) after they happen.
+
+## Design decisions
+
+The above requirements led us to the following design:
+
+**Each node exposes its own metrics in Prometheus format**
+
+Prometheus is an industry-standard format, and each node exposing its own metrics allows anyone to monitor their own node directly.
+
+**Additionally, each node is able to retrieve any other connected node’s metrics and expose them in Prometheus format**
+
+This enables not only Protocol Labs but any LoE member to create their own dashboards which show the state of the whole network.
+
+**The transport of metrics between nodes is done using the same mechanism that drand uses for inter-node communication when generating randomness (GRPC)**
+
+With this we allow metrics to be sent between nodes, while not requiring opening any additional network ports.
```

Comment:
```suggestion
With this we allow metrics to be sent between nodes, without requiring to open any additional network port.
```

Reply:
disagree with both 'to open' and 'port'

Reply:
You're not opening any, so why plural?

Reply:
I don't know why, it's just right ;p

Reply:
Plural "feels" more right. I also disagree with the change, `opening` is correct.